### PR TITLE
Features/gp 3 embree support precompiled libs

### PR DESCRIPTION
**Pull request by sentik**
Added support for SIMD & embree support precompiled libs

### CODE REVIEW

**Comment by Calinou** [comment]
Thanks for opening a pull request :slightly_smiling_face: 

Feature pull requests should be associated to a *feature proposal* to justify the need for implementing the feature in Godot core. Please open a proposal on the [Godot proposals](https://github.com/godotengine/godot-proposals) repository (and link to this pull request in the proposal's body).

Also, note that we do not use Git submodules (and have no intent to), due to their poor UX.

**Comment by fire** [comment]
I also remember that due to our userbase anything simd above sse4.1? has problems.

File: 1_build.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"scons" p=windows target=editor arch="x86_64" debug_symbols="yes" builtin_embree="no" use_llvm="yes" optimize="speed_trace" vsproj="yes" compiledb="yes" openxr="no" opengl3="no" deprecated="no" module_arkit_enabled="no" module_mobile_vr_enabled="no" module_mono_enabled="no" module_webxr_enabled="no" module_webrtc_enabled="no" module_websocket_enabled="no" -j 64
```
Comment (aaronfranke) [inline 1_build.bat:1]: This `.bat` file should not be here.

**Comment by Calinou** [comment]
> I also remember that due to our userbase anything simd above sse4? has problems.

If dynamic dispatch is used, it's not a problem. However, binary size will be impacted as all variants need to be compiled in the binary (from SSE to AVX2).

**Review by aaronfranke — COMMENTED** [review]
Godot does not use submodules, so this PR can't be accepted until the submodule is removed.

We also probably don't want to only have the library precompiled, there are several disadvantages to that.

**Comment by sentik** [comment]
sorry I submitted a PR to the wrong repository. i wanted to make changes and push them in my fork instead of public. I'll close this PR for now.

**Pull request closed without merging**